Source/JavaScriptCore/wasm/WasmTypeDefinition.h

```diff
@@ -876,6 +894,8 @@ class TypeInformation {
     RefPtr<TypeDefinition> m_Void_I32I32I32I32;
     RefPtr<TypeDefinition> m_Void_I32I32I32I32I32;
     RefPtr<TypeDefinition> m_I32_I32;
+    RefPtr<TypeDefinition> m_I32_RefI32I32;
```

Comment:
Note: the style checker complains about these class members, but they are named to be consistent with the other types above.